### ISSUE TITLE
MDAE (Minima Decentralized Advertising Ecosytem) - Josua's Team

### ISSUE THREAD

**Issue by MDAE-DAO** [minima-global/innovation-challenge#8]
**Before you start, please update the issue title:**

Example: *MDAE (Minima Decentralized Advertising Ecosytem) - Josua's Team*

## :pencil2: Project & Team Details

1. MiniDapp/Project Name [required, as used in your Google Form]: *MDAE (Minima Decentralized Advertising Ecosytem)*
[MDAE_Pitch_ Presentation.pdf](https://github.com/minima-global/innovation-challenge/files/9259527/MDAE_Pitch_.Presentation.pdf)
[MDAE White Paper v0.1.pdf](https://github.com/minima-global/innovation-challenge/files/9259528/MDAE.White.Paper.v0.1.pdf)
*

2. MiniDapp/Project Description [optional, if changed from your application in Google Form]: *Please provide a brief description of your MiniDapp/Project*

3. Team Name [optional, if changed from your application in Google Form]: *Provide your team name*

4. Team Members [optional, if changed from your application in Google Form]: *Please list names of all team members*

## :one: Phase 1 Submission 

For Phase 1, please upload the following 
(paste the link to where you uploaded e.g. other cloud provider or GitHub (if more than 100MB) or Attach files by dragging & dropping, selecting or pasting them)

5. A whitepaper or a pitch video with a description of the idea you have that includes: your idea, how it meets the criteria to be considered for the competition, the benefits it will bring to people and how the unique features of the Minima network can make a real difference/give an advantage to your solution.

6. Upload Presentation: *Upload any relevant presentations/slide decks you have created*

## :two: Phase 2 Submission 

For Phase 2, please update your submission, providing the following:

7. Code Repository Link: *Provide a link to the Github URL (or other) for your MiniDapp/Project*

8. Upload MiniDapp: *If your project includes a MiniDapp, please upload your MiniDapp - this should be a .zip file using the **.mds.zip** extension*

9. Upload Demo: *Upload any demonstration videos you have created (optional)*

##  :round_pushpin: Other Comments
*Optional - add any other comments regarding your submission*

## :heavy_check_mark: Checklist

**Before you post the issue**:
- [ ] Make sure you have entered using this [Google Form](https://docs.google.com/forms/d/e/1FAIpQLSdW-r1iTN2JUxaLhKZxLj4FgRnIF6yZWAXB30hj4c-vwaNuPw/viewform)
- [ ] You have followed the issue title format
- [ ] You have provided all the information correctly and uploaded all required material for the Phase you are submitting for
- [ ] You have read and agree with the terms in the [Code Of Conduct]()

**Comment by MDAE-DAO** [comment]
Well, I don't know it I did it right or not. 
Can somebody notify if it is all OK ?

**Comment by martapiekarska** [comment]
Looks amazing!

**Comment by MDAE-DAO** [comment]
Thank you , what a relief, I wasn't sure it was posted right :-)

**Comment by martapiekarska** [comment]
Hi Team! Can I please ask you to confirm that you are working on your final submission - the final deadline is 12th of September. Please try taking the continuous submission approach rather than send everything at once. This will help judges and show the journey you’re taking and the thought process.

**Comment by MDAE-DAO** [comment]
Hi Marta,

yes we are working on the projects, but we have an important delay, as I told you the dead line was too tight, and we can't do continuous submission, as the code is continuously changing as we are evolving.

Sorry for the delay, but I was using other email until today I opened this one.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by martapiekarska** [comment]
Hi the deadline has been moved to 12th of September, as per your request. We cannot move it any further than that.

**Comment by MDAE-DAO** [comment]
yes, I know it can be moved :-( but it is what it is, we do the best we can with all problems that are arising, so playing with scripts and transactions it is very complex as the error messages don't help, can't be anythig and you have to spent hours trying to figure it out what is failing.

thanks.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by MDAE-DAO** [comment]
I can't submit the project, did what it says, edit you issue and set it up all info asked, but only prompts de name of the project and save option.

**Comment by martapiekarska** [comment]
Please send all the files to ***@***.***
